index.html

```diff
@@ -11,30 +11,190 @@
+               <ul class="lists1">
+                <li>Css</li>
+                <li>html</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+               </ul>
+               <button>See Project</button>
+            </div>
+            <div class="work2">
+            <h5>Professional Art Printing Data</h5>
+            <p>A daily selection of privetly personalized reads; no accounts or sign-ups required.
+              has been the industry's standard.</p>
+              <ul class="lists">
+                <li>html</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+               </ul>
+               <button>See project</button>
+            </div>
+            <div class="work2">
+            <h5>Professional Art Printing Data</h5>
+            <p>A daily selection of privetly personalized reads; no accounts or sign-ups required.
+              has been the industry's standard.</p>
+              <ul class="lists">
+                <li>html</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+               </ul>
+            <button>See project</button>
+            </div>
+            <div class="work2">
+            <h5>Professional Art Printing Data</h5>
+            <p>A daily selection of privetly personalized reads; no accounts or sign-ups required.
+              has been the industry's standard.</p>
+              <ul class="lists">
+                <li>html</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+               </ul>
+               <button>See project</button>
+            </div>
+            <div class="work2">
+            <h5>Professional Art Printing Data</h5>
+            <p>A daily selection of privetly personalized reads; no accounts or sign-ups required.
+              has been the industry's standard.</p>
+              <ul class="lists">
+                <li>html</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+               </ul>
+               <button>See project</button>
+            </div>
+            <div class="work2">
+            <h5>Professional Art Printing Data</h5>
+            <p>A daily selection of privetly personalized reads; no accounts or sign-ups required.
+              has been the industry's standard.</p>
+              <ul class="lists">
+                <li>html</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+               </ul>
+               <button>See project</button>
+            </div>
+            <div class="work2">
+            <h5>Professional Art Printing Data</h5>
+            <p>A daily selection of privetly personalized reads; no accounts or sign-ups required.
+              has been the industry's standard.</p>
+              <ul class="lists">
+                <li>html</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+               </ul>
+               <button>See project</button>
+            </div>
```

Comment:
As shown in the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27), please add the respective images to each project card. Also, because of this any of the content inside the cards is not showing (title, description, etc) since the background is white and so is the text inside :exclamation:  :100: